src/cephadm/cephadm

```diff
@@ -3394,10 +3394,14 @@ if __name__ == "__main__":
     try:
         r = args.func()
     except Error as e:
+        logger.exception('execution failed')
```

Comment:
This adds a traceback to the console in all `Error` cases, which is what I though we were trying to avoid ... ?

Reply:
right. tried to add this just to the log file, not the console

Reply:
Do we need two loggers? one console+file logger and a dedicated console-only logger?